coffee/lib/abstract-chosen.coffee

```diff
@@ -27,6 +27,7 @@ class AbstractChosen
     @enable_split_word_search = if @options.enable_split_word_search? then @options.enable_split_word_search else true
     @group_search = if @options.group_search? then @options.group_search else true
     @search_contains = @options.search_contains || false
+    @search_word_boundary = if @options.search_word_boundary? then @options.search_word_boundary else '^|\\s|\\b'
```

Comment:
Can’t this be just;

```coffee
@search_word_boundary = options.search_word_boundary || '^|\\s|\\b'
``` 

?

Reply:
Also, the option should not necessarily include “search” in its name

Reply:
Is it not logical to prefix search related options with "search"?

And I'm pretty sure the question mark is needed, otherwise the empty string will evaluate to false and the default value will be used. The empty string is a perfectly valid value for this option... if set to an empty string Chosen works like `search_contains=true`, but allowing `enable_split_word_search` to decide whether to only match the beginning of the option - see linked issue for a relevant use case.